ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    if not os.path.exists("/opt/deeplearning/metadata/env_version"):
</pre><p>Check the notebook_template for an updated cell showing IS_GOOGLE_CLOUD_NOTEBOOK var.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>